packages/components/organisms/f-checkout/src/store/tests/checkout.module.test.js

```diff
@@ -603,6 +604,7 @@ describe('CheckoutModule', () => {
         describe('getAddress ::', () => {
             it(`should get the address details from the backend and call ${UPDATE_FULFILMENT_ADDRESS} mutation.`, async () => {
                 // Arrange
+                payload = { ...payload, currentPostcode: 'JZ11AA' };
```

Comment:
Could we give this magic string a name?  If it's just any valid postcode then putting it in a const called something like `DUMMY_POSTCODE` or `VALID_POSTCODE`.